editor/html-template.ts

```diff
@@ -1,24 +1,19 @@
 import { version as goslingVersion } from '../package.json';
-const higlassVersion = '1.11';
-const reactVersion = '17';
-const pixiVersion = '6';
+import { version as higlassVersion } from 'higlass/package.json';
 
 export const getHtmlTemplate = (spec: string) => `
 <!DOCTYPE html>
 <html>
 <head>
     <title>Gosling Visualization</title>
-    <link rel="stylesheet" href="https://unpkg.com/higlass@${higlassVersion}/dist/hglib.css">
-    <script src="https://unpkg.com/react@${reactVersion}/umd/react.production.min.js"></script>
-    <script src="https://unpkg.com/react-dom@${reactVersion}/umd/react-dom.production.min.js"></script>
-    <script src="https://unpkg.com/pixi.js@${pixiVersion}/dist/browser/pixi.min.js"></script>
-    <script src="https://unpkg.com/higlass@${higlassVersion}/dist/hglib.js"></script>
-    <script src="https://unpkg.com/gosling.js@${goslingVersion}/dist/gosling.js"></script>
+    <link rel="stylesheet" href="https://esm.sh/higlass@${higlassVersion}/dist/hglib.css">
+    
 </head>
 <body>
-    <div id="gosling-container"/>
-    <script>
-        gosling.embed(document.getElementById('gosling-container'), ${spec})
+    <div id="gosling-container"></div>
+    <script type="module">
+        import { embed } from 'https://esm.sh/gosling.js@${goslingVersion}?bundle';
```

Comment:
The only question I have here is whether the version of higlass and gosling.js will stay in sync over time. Unfortunately the names used in the styles are specific to the higlass version, so I don't know if there could be a case where higlass makes a patch (e.g., v1.11.8) but we have styles from 1.11.7 in our template and the two end up out of sync. I wonder if it's possible to do something like:

```javascript
        import { embed } from 'https://esm.sh/gosling.js@${goslingVersion}?bundle&deps=higlass@${higlassVersion}';
```

but I'm not sure if the `deps` is handled when `bundle` is used.

Reply:
Is it possible to use different react and react-dom versions as well that user wishes to use?

Reply:
Same question. Ideally you could bundle _and_ override dependency versions, but I'm not sure for ESM.sh if that is current possible.

Reply:
Just tested. Unfortunately, it seems to ignore the version of the dependencies. I tried using a lower version of HiGlass and I still got `HiGlass v1.12.4: http://higlass.io/` in the console. What I'll try next is to use the pre-bundled peer dependencies in an import map. 

```javascript
import { embed } from 'https://esm.sh/gosling.js@0.11.0?bundle&deps=higlass@$1.11.7';
```

Reply:
I didn't have any luck using unbundled gosling.js with bundled peer dependencies either using an importmap. 

The problem seems to be that HiGlass has `"pixi.js": "^5.0.3"` as a peer dependency. One of the pixi subpackages  called `@pixi/polyfill` relies on a package called `es6-promise-polyfill` which doesn't have the export that @pixi/polyfill expects. Thus I get a `The requested module '/v133/es6-promise-polyfill@1.2.0/es2022/es6-promise-polyfill.mjs' does not provide an export named 'Polyfill'` error when trying to import HiGlass. 

Looking at that version of es6-promise-pollyfill it does look like there is export called [Polyfill](https://github.com/lahmatiy/es6-promise-polyfill/blob/v1.2.0/promise.js#L29-L50) but I don't think it gets converted in a way that is accessible in the [ESM version](https://esm.sh/v133/es6-promise-polyfill@1.2.0/es2022/es6-promise-polyfill.mjs) that esm.sh generates. 

How hard it would it be to upgrade the pixi version that HiGlass uses to v6?

Reply:
Added a PR to HiGlass to upgrade Pixi https://github.com/gosling-lang/gosling.js/pull/977